routers/custom_router.py

```
@@ -0,0 +1,32 @@
+    input_name: NameReqDTO, handler: NameHandler = Depends(get_name_handler)
+):
+    return handler.create_name(input_name)
+
+
+@custom_router.get("/")
+def get_names(handler: NameHandler = Depends(get_name_handler)):
+    return handler.get_names()
+
+
+@custom_router.delete("/{index}")
+def delete_name(index: int, handler: NameHandler = Depends(get_name_handler)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)